modules/virt-attaching-vm-secondary-network-cli.adoc

```diff
@@ -37,17 +37,20 @@ spec:
       networks:
         - name: default
           pod: {}
-        - name: bridge-net <1>
+        - name: bridge-net <2>
           multus:
-            networkName: <network-namespace/a-bridge-network> <2>
+            networkName: networkName: <network-namespace>/<a-bridge-network> <3>
 ...
 ----
-<1> The name of the bridge interface. This value must match the `name` value of the corresponding `spec.template.spec.networks` entry.
-<2> The name of the network attachment definition.
+<1> The name of the bridge interface.
+<2> The name of the bridge interface. This value must match the `name` value of the corresponding `spec.template.spec.networks` entry.
+<3> The namespace where the Network Attachment Definition exists. The only 2 possible options for this namespace are either the default namespace or the same namespace where the VM is to be created.
 +
 [NOTE]
 ====
-The example virtual machine is connected to both the `default` pod network and `bridge-net`, which is defined by a network attachment definition named `a-bridge-network`. If the network attachment definition is not in the same namespace as the VM, you must combine the namespace and the network name to create the network attachment definition name.
+The example virtual machine is connected to both the default pod network and the secondary network defined by a network attachment definition (NAD). The NAD must be created either in the same namespace as the VM or in the `default` namespace. However, if the current working namespace is different than the one where the NAD is created, the NAD name must be prefixed with the NAD namespace such `default/a-bridge-network` (as in any other namespace resources in Openshift).
```

Comment:
_as in any other namespace resources in Openshift_
"namespace resources" is confusing, I think. It implies to resources of `kind: Namespace`, whereas this sentence should refer to **namespaced** resources, which are resources that exist only in a specific namespace (note that "namespaced resources" is the phrase we use, but I am not sure it is the official phrasing that can be used in the user's documentation).

Reply:
Can I remove the parenthetical part altogether? Or is it essential?

Reply:
Yes, I think you can remove it. It's clear without it.

Reply:
Ok, I've removed the parenthetical and fixed the duplication. Hopefully all is well now.